api/segmentation/class-segmentation.php

```diff
@@ -12,11 +12,9 @@
  */
 class Segmentation {
 	/**
-	 * Initialize.
+	 * The log file path.
 	 */
-	public static function init() {
-		self::create_segmentation_tables();
-	}
+	const LOG_FILE_PATH = '/tmp/newspack-popups-events.log';
```

Comment:
I'm not sure we can reliably assume the existence of this directory. Also, this could create chaos in situations where there are multiple WordPress installations hosted on the same server, and multisite too. I think we need to [programmatically get the path to the log file](https://stackoverflow.com/questions/8955411/find-out-the-error-logs-path), then add elements to the filename to ensure uniqueness. I think the database prefix will do the trick. As for the multisite scenario, I'm not sure how best to address.

Reply:
I've used [WP function `get_temp_dir`](https://developer.wordpress.org/reference/functions/get_temp_dir/) in f66afff4ed2a4fe91091fee9a3895e3f69bbc298 – does that look ok?